mangaki/mangaki/views.py

```diff
@@ -711,3 +712,20 @@ def faq_index(request):
         'information': all_information,
     }
     return render(request, 'faq/faq_index.html', context)
+
+def generic_error_view(error, error_code):
+    def error_view(request):
+        try:
+            trope = Trope.objects.order_by('?').first()
+        except DatabaseError:
+            return server_error
```

Comment:
Il faut renvoyer `server_error(request)`, d'après la doc : https://docs.djangoproject.com/fr/1.10/ref/views/#the-500-server-error-view